test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -391,7 +392,7 @@ def setUp(self):
 sleep 300
 """
         self.cfg0 = """{
-    "cgroup_prefix"         : "pbspro",
+    "cgroup_prefix"         : "pbs",
```

Comment:
Please also check for instances of "propbs" in this file and update them accordingly. I believe the intent was to ensure the cgroup_prefix could be changed.